src/libcore/raw.rs

```diff
@@ -6,7 +6,7 @@
 //! They can be used as targets of transmutes in unsafe code for manipulating
 //! the raw representations directly.
 //!
-//! Their definition should always match the ABI defined in `rustc::back::abi`.
+//! Their definition should always match the ABI defined in `rustc_target::abi`.
```

Comment:
`rustc_middle::ty::layout` (which IMO should be `rustc_middle::ty::abi` or `rustc_middle::abi` but I've never gotten around to it)